lib/prerequisite.js

```diff
@@ -59,10 +60,9 @@ module.exports = (input, pkg, options) => {
 		{
 			title: 'Check git remote',
 			task: async () => {
-				try {
-					await execa('git', ['ls-remote', 'origin', 'HEAD']);
-				} catch (error) {
-					throw new Error(error.stderr.replace('fatal:', 'Git fatal error:'));
+				const valid = await validRemote();
```

Comment:
The naming of the constant and the usage of `validRemote` here may be a bit unclear (because of https://github.com/sindresorhus/np/pull/326/files#r247307935). I couldn't come up with a better name, but if necessary I can add a comment here.